src/DeviceInput/Implementations/webDeviceInputSystem.ts

```diff
@@ -41,6 +41,8 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
     private _pointerBlurEvent = (evt: any) => { };
     private _wheelEventName: string;
 
+    private _mouseId = -1;

```

Comment:
I think we may want to track a list of buttons we have captured, rather than assuming that only one pointer is pressed. If the user presses multiple mouse buttons down, we should capture them all, and then release them all when the related pointerup event occurs.

Reply:
Capturing the pointer should capture all pointer events to the element. I don't think we need to worry about which buttons. They will all be captured. If there are multiple mouse devices connected, then we may have an issue.

Reply:
For pointer button events, a "pointerup" event is only thrown all buttons are up.  Otherwise, a "pointermove" event is thrown when a button is released.

Things do get a little tricky when the cursor leaves the canvas with multiple buttons pressed though.  There is a standing bug (since 4.0) where dragging will "stick" if the cursor leaves the canvas with multiple buttons pressed.  It may be worth fleshing out the logic to try and fix this, though I might relegate that to a different PR so that Texture Inspector gets fixed first.